spring/src/main/java/com/connexta/transformation/rest/spring/TransformApiRetrieve.java

```diff
@@ -45,7 +50,25 @@
       })
   @ApiResponses(
       value = {
-        @ApiResponse(code = 200, message = "Successfully retrieved.", response = Resource.class),
+        @ApiResponse(
+            code = 200,
+            message = "Successfully retrieved.",
+            response = Resource.class,
+            responseHeaders = {
+              @ResponseHeader(
+                  name = "Last-Modified",
+                  description =
+                      "The time at which this piece of metadata finished being generated.",
```

Comment:
❓ Should this align a bit more with our terminology? It's not necessarily metadata, it could be a thumbnail, for example.

Reply:
It will be once we revised the API. For now, this is still the same API as before.